rmgpy/molecule/molecule.py

```diff
@@ -60,6 +60,12 @@
 
 ################################################################################
 
+bond_orders = {'S': 1, 'D': 2, 'T': 3}
+
+globals().update({
+    'bond_orders': bond_orders,
```

Comment:
Just for the sake of me learning, what's the purpose of updating the `globals()` dictionary?